.github/workflows/golangci-lint.yml

```diff
@@ -14,8 +14,15 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
```

Comment:
Isn't this required for those nice inline comments?

Reply:
I believe so. Did you test this @mindaugasrukas ?